makejdk.sh

```diff
@@ -120,11 +122,23 @@ parseCommandLineArgs()
       "--freetype-dir" | "-ftd" )
       export FREETYPE_DIRECTORY="$1"; shift;;
 
+      "--variant"  | "-bv" )
+      export BUILD_VARIANT=$1; shift;;
+
       *) echo >&2 "${error}Invalid option: ${opt}${normal}"; man ./makejdk-any-platform.1; exit 1;;
      esac
   done
 }
 
+doAnyBuildVariantOverrides()
+{
+  if [[ "${BUILD_VARIANT}" == "openj9" ]]; then
```

Comment:
Is this `BUILD_VARIANT` overlapping with OpenJDK's own variant terminologies (borderline)?

Reply:
No, it's separate because OpenJ9's build system does not (yet) rely on OpenJDK's variant stuff (sorry, we know that's not good). Our plan is to fix that, but for now we needed this extra little bit. My hope is that, in future, we can remove the option and change BUILD_VARIANT to JVM_VARIANT.

Reply:
Ok cool thnx